src/System.Text.Json/tests/Serialization/TestClasses.cs

```diff
@@ -754,6 +755,295 @@ public void Verify()
         }
     }
 
+    public class TestClassWithObjectIEnumerableConstructibleTypes : ITestClass
```

Comment:
nit: the tests seem to have a lot of copy/paste redundancy that could be cleaned up with some refactoring

Reply:
Yes, I'll address this in https://github.com/dotnet/corefx/issues/37520.